<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1020</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Multilayer Perceptron</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(80, 80, 80);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_open_dataset">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>100</y>
      <width>101</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  color: white;
  background-color: rgb(200, 150, 50);
  border: 1px solid black;
}
QPushButton:pressed {
    background-color: rgb(150, 100, 0);
}</string>
    </property>
    <property name="text">
     <string>Open dataset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_open_model">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>20</y>
      <width>101</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  color: white;
  background-color: rgb(200, 150, 50);
  border: 1px solid black;
}
QPushButton:pressed {
    background-color: rgb(150, 100, 0);
}</string>
    </property>
    <property name="text">
     <string>Open model</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="matrix_model">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>280</y>
      <width>99</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>matrix</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="graph_model">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>310</y>
      <width>99</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>graph</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_model">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>250</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>Model:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_open_bmp">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>510</y>
      <width>101</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  color: white;
  background-color: rgb(200, 150, 50);
  border: 1px solid black;
}
QPushButton:pressed {
    background-color: rgb(150, 100, 0);
}</string>
    </property>
    <property name="text">
     <string>Open BMP</string>
    </property>
   </widget>
   <widget class="QLabel" name="slider_dataset_part">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>180</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>Part of dataset:</string>
    </property>
   </widget>
   <widget class="QSlider" name="test_part">
    <property name="geometry">
     <rect>
      <x>839</x>
      <y>220</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="sliderPosition">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_save_model">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>60</y>
      <width>101</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  color: white;
  background-color: rgb(200, 150, 50);
  border: 1px solid black;
}
QPushButton:pressed {
    background-color: rgb(150, 100, 0);
}</string>
    </property>
    <property name="text">
     <string>Save model</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>340</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="layers_layout">
     <item>
      <widget class="QLabel" name="label_hiden_layers">
       <property name="styleSheet">
        <string notr="true">color: white;</string>
       </property>
       <property name="text">
        <string>Hiden layers:</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignRight">
      <widget class="QSpinBox" name="hiden_layers_number">
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="maximum">
        <number>5</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>380</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="epoch_layout">
     <item>
      <widget class="QLabel" name="label_epoch">
       <property name="styleSheet">
        <string notr="true">color: white;</string>
       </property>
       <property name="text">
        <string>Epoch:</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignRight">
      <widget class="QSpinBox" name="epoch_number">
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="maximum">
        <number>5</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>70</y>
      <width>512</width>
      <height>512</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>512</width>
      <height>512</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_train_model">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>430</y>
      <width>101</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  color: white;
  background-color: rgb(200, 150, 50);
  border: 1px solid black;
}
QPushButton:pressed {
    background-color: rgb(150, 100, 0);
}</string>
    </property>
    <property name="text">
     <string>Train model</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_test_model">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>470</y>
      <width>101</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  color: white;
  background-color: rgb(200, 150, 50);
  border: 1px solid black;
}
QPushButton:pressed {
    background-color: rgb(150, 100, 0);
}</string>
    </property>
    <property name="text">
     <string>Test model</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_recognize_image">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>550</y>
      <width>101</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  color: white;
  background-color: rgb(200, 150, 50);
  border: 1px solid black;
}
QPushButton:pressed {
    background-color: rgb(150, 100, 0);
}</string>
    </property>
    <property name="text">
     <string>Recognize</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clear_paint">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>590</y>
      <width>101</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  color: white;
  background-color: rgb(200, 150, 50);
  border: 1px solid black;
}
QPushButton:pressed {
    background-color: rgb(150, 100, 0);
}</string>
    </property>
    <property name="text">
     <string>Clear paint</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_letter">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>600</y>
      <width>61</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>Probably:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_recognized_letter">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>600</y>
      <width>41</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_open_test_dataset">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>140</y>
      <width>101</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  color: white;
  background-color: rgb(200, 150, 50);
  border: 1px solid black;
}
QPushButton:pressed {
    background-color: rgb(150, 100, 0);
}</string>
    </property>
    <property name="text">
     <string>Open test DS</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_test_epoch">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>650</y>
      <width>131</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>Test results for epoch:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_test_error">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>680</y>
      <width>61</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>Error:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_test_error_val">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>680</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_test_epoch_val">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>650</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
